sub-storage/src/lib.rs

```diff
@@ -175,6 +178,98 @@ where
 		.collect::<Result<Vec<(K, V)>, &'static str>>()
 }
 
+/// Read from keys regardless of the type.
+pub async fn batchRead(keys: Vec<StorageKey>, client: &Client, at: Hash) -> Vec<(StorageKey, StorageData)> {
```

Comment:
```suggestion
pub async fn batch_read(keys: Vec<StorageKey>, client: &Client, at: Hash) -> Vec<(StorageKey, StorageData)> {
```


ewww that looks more like js style :p